### ISSUE TITLE
Possible support Vue.js or Nuxt.js ?

### ISSUE THREAD

**Issue by krittin-58** [smplrspace/docs#26]
It's be good if support multi framework. 

Thanks.

**Comment by tibotiber** [comment]
Yes, this is in the roadmap. Not a big priority however so you might have to be patient.

By the way, we're not using issues – I didn't realise they were even active. I will be deactivating issues completely. Would you mind reaching out via email at support@smplrspace.com so we know your email and can contact you regarding this with any progress?

Thanks.
